### PR TITLE
etcd: Update to v3.4.9

### PR DESCRIPTION
**Pull request by surajssd**
Release Notes: https://github.com/etcd-io/etcd/releases/tag/v3.4.9

### CODE REVIEW

File: assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl [hunk 1]
```diff
@@ -7,7 +7,7 @@ systemd:
         - name: 40-etcd-cluster.conf
           contents: |
             [Service]
-            Environment="ETCD_IMAGE_TAG=v3.4.7"
+            Environment="ETCD_IMAGE_TAG=v3.4.9"
```
Comment (ipochi) [inline assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl:10]: Should we wait for the upstream kubernetes to bump up its etcd version to 3.4.9 before we do ? 
https://github.com/kubernetes/kubernetes/issues/91266

If its something that can wait then lets wait.

Comment (surajssd) [reply assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl:10]: Sure we can wait.

Comment (invidian) [reply assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl:10]: I think it's fine to update, we get fixes quicker :)

Comment (iaguis) [reply assets/lokomotive-kubernetes/aws/flatcar-linux/kubernetes/cl/controller.yaml.tmpl:10]: or bugs :see_no_evil:

**Comment by invidian** [comment]
Can we merge this PR?

**Pull request merged**